README.md

```diff
@@ -17,6 +18,11 @@ Secondary zones allow client machine in primary DNS zones to do DNS resolution o
 * **xDnsServerZoneTransfer** This resource allows a DNS Server zone data to be replicated to another DNS server.
 * **xDnsRecord** This resource allwos for the creation of IPv4 host (A) records or CNames against a specific zone on the DNS server
 
+### xDnsServerForwarder
+
+* **IPAddress**: IP address of the forwarder
+* **RemoveAll**: Whether all forwarders should be removed or not
+* **Ensure**: Whether the forwarder should be present or removed
 
 ### xDnsServerPrimaryZone
 
```

Comment:
Please include a summary of changes in an unreleased version change per the [contribution guidelines](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#lifecycle-of-a-pull-request)